src/AffineTransform/index.ts

```diff
@@ -1,92 +1,89 @@
-import { right, left, Either } from 'fp-ts/lib/Either';
+import { right, left } from 'fp-ts/lib/Either';
 
 import { IAffineTransform } from './interface';
 import { SquareMatrix3D, Matrix } from '../Matrix';
 import { IVector, Vector } from '../Vector';
-import { IPosition3D as IPos3D, Position3D } from '../Position';
+import { Position3D, IPosition3D } from '../Position';
 import { AxisAlignedBoundingBox } from '../AxisAlignedBoundingBox';
 import { CubeCorners } from '../Cube';
 import { Pair } from '../util/Pair';
+import { FineResult } from '../interface';
 
 export { IAffineTransform };
 
 // TODO: export this to AffineTransform.ts
 export class AffineTransform3D implements IAffineTransform {
-    /**
```

Comment:
There are some format inconsistencies (2-space tabs instead of 4) that are bringing more changes than intended.

Reply:
I've added a .prettierrc and formatted those with 2-spaces to have 4